apps/server/src/modules/board/controller/dto/card/create-card.body.params.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ContentElementType } from '@shared/domain';
+import { IsEnum } from 'class-validator';
+
+export class CreateCardBodyParams {
+	@IsEnum(ContentElementType, { each: true })
+	@ApiProperty({
+		required: false,
+		isArray: true,
+		enum: ContentElementType,
+	})
+	requiredEmptyElements?: ContentElementType[];
```

Comment:
I don't understand the naming 'requiredEmptyElements'
this is very similar to CreateContentElementBody in create-content-element.body.ts, except it can take an array and can is optional, so at least the naming should be more like that

Moreover, I wonder if we can simply use and array of CreateContentElementBody[] instead of ContentElementType

Reply:
@virgilchirial 
My thoughts about your suggestion using CreateContentElementBody[]
Advantage: if there is a change in the requirements to create a ContentElement it can be seen directly in CreateCardBodyParam - DTO
Disadvantage: the api is more complicated - instead of having an array with simple strings we have an array of objects that have to be destructed.
Just to have an array of the types were at least the wish of north star. But I am not sure...

Reply:
Regarding the name: the purpose of this parameter is to create empty content elements of given type if it's required by the frontend -> requiredEmptyElements
CreateContentElementBody is only the purpose of each index of this array. This in turn leads to your consideration using the CreateContentElementBody[] -> difficult

Reply:
we discussed it. It remains as it is for now